condarecipe/larray/larray-editor-script.pyw

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+if __name__ == '__main__':
+    import os
+    import sys
+    from larray import edit
+
+    if sys.executable.endswith("pythonw.exe"):
+        sys.stdout = open(os.devnull, "w")
+        sys.stderr = open(os.path.join(os.getenv("TEMP"), "stderr-" + os.path.basename(sys.argv[0])), "w")
+
+    sys.exit(edit(-2))
```

Comment:
I am unsure we should have this "open last" by default. Opening some files can take a while and it would be frustrating if it wasn't the file they wanted to open. This would be a nice option when we will have a "Preferences" dialog within the viewer but until then I wouldn't do it.